packages/marko-web-inquiry/env.js

```diff
@@ -3,6 +3,9 @@ const { cleanEnv, validators, str } = require('@base-cms/env');
 const { nonemptystr } = validators;
 
 module.exports = cleanEnv(process.env, {
+  // these are the google recaptcha test keys
+  RECAPTCHA_SITE_KEY: str({ desc: 'An site key for sending recaptcha validation.', default: '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI' }),
+  RECAPTCHA_SECRET_KEY: str({ desc: 'A secret key for sending recaptcha validation.', default: '6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe' }),
```

Comment:
@B77Mills The secret key cannot include a default value. Also, this key must now be immediately invalidated and new keys rotated in wherever it was used, as this is a public commit.

Reply:
@solocommand  These are the google dev keys